src/store/types.ts

```diff
@@ -0,0 +1,3 @@
+export const ADD_USER_INFO = "ADD_USER_INFO";
+export const SET_REDUX_KEY = "SET_REDUX_KEY";
+export const API_DATA="API_DATA";
```

Comment:
Name it ADD_API_DATA as we are using it to add api data into redux

Reply:
done